app/assets/src/components/views/discovery/mapping/map_tooltip.scss

```diff
@@ -0,0 +1,55 @@
+@import "~styles/themes/colors";
+@import "~styles/themes/typography";
+
+.mapTooltip {
+  z-index: 1;
+
+  // Global to override some MapboxGL styling
+  :global(.mapboxgl-popup-content) {
+    $tooltip-border-width: 1px;
```

Comment:
Nit: Could we move constants to the top of the file, below the imports?

Reply:
👍 done